src/NServiceBus.RavenDB/Subscriptions/RavenDbSubscriptionStorage.cs

```diff
@@ -13,26 +12,42 @@ class RavenDbSubscriptionStorage : Feature
 
         protected override void Setup(FeatureConfigurationContext context)
         {
-            context.Container.ConfigureComponent<ISubscriptionStorage>(b =>
+            var disableAggressiveCaching = context.Settings.GetOrDefault<bool>(DoNotAggressivelyCacheSubscriptionsSettingsKey);
+            if (!context.Settings.TryGet(AggressiveCacheDurationSettingsKey, out TimeSpan aggressiveCacheDuration))
             {
-                var store = DocumentStoreManager.GetDocumentStore<StorageType.Subscriptions>(context.Settings, b);
-
-                var persister = new SubscriptionPersister(store);
+                aggressiveCacheDuration = TimeSpan.FromMinutes(1);
```

Comment:
Default that was previously used inside the property getter of `SubscriptionPersister`